docs/content/querying/segmentmetadataquery.md

```diff
@@ -52,6 +52,13 @@ Timestamp column will have type `LONG`.
 
 Only columns which are dimensions (ie, have type `STRING`) will have any cardinality. Rest of the columns (timestamp and metric columns) will show cardinality as `null`.
 
+### intervals
+
+If an interval is not specified, the query will use a default interval that spans a configurable period before the end time of the most recent segment.
+
+The length of this default time period is set in the broker configuration via:
+  druid.query.segmentMetadata.defaultSegmentHistoryPeriod
```

Comment:
I believe the broker currently also hardcodes one week as well, so it would be nice to make it a common config for both to be consistent, since those two are kind of related.